backstop/tests/color-picker.html

```diff
@@ -0,0 +1,271 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Color Picker | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <h1>Color Picker</h1>
+    <hr />
+
+    <h2>Basic</h2>
+    <section id="demo-basic">
+      <div class="iui-color-picker">
+        <div class="iui-color-palette">
```

Comment:
I don't think you need two divs here.

Reply:
I think it does make sense because it is 1 part within the advanced color picker.  Might be able to exclude the div in the basic color picker if we use mixins tho.